src/models/user-played-time-response.model.ts

```diff
@@ -0,0 +1,44 @@
+import type { GameConcept } from "./title-concept.model";
+import type { Media } from "./game-media.model";
+
+export interface GetUserPlayedTimeResponse {
+  /** The title ID of this game 
+      @example CUSA26431_00
+      */
+  titleId: string;
+
+  /** Default name for the title */
+  name: string;
+
+  /** Localized name for the title */
+  localizedName: string;
+
+  /** URL to game's console launch screen, from image.api.playstation.com */
```

Comment:
"from X" can be cut. Much more valuable would be an @example, which would demonstrate this.